production-swarm/docker-stack.yml

```diff
@@ -0,0 +1,58 @@
+version: '3'
+services:
+  jobservice:
+    env_file:
+      - ./rabbitmq.env
+      - ./postgres.env
+      - ./jobservice.env
+    environment:
+      CAF_DATABASE_USERNAME: "${CAF_DATABASE_USERNAME:-postgres}"
```

Comment:
Not sure what to do here as some can set this by change the "postgres", by replacing the entire item "${CAF_DATABASE_USERNAME:-postgres}" or by setting an environment var